### PR TITLE
Apply upstream changes. r=Yoric

### PR DESCRIPTION
**Pull request by ferjm**
This depends on https://github.com/fxbox/taxonomy/pull/55

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ serde = "0.6.13"
 serde_json = "0.6.0"
 serde_macros = "0.6.14"
 chrono = "0.2.19"
-foxbox_taxonomy = { git = "https://github.com/fxbox/taxonomy.git", rev = "ac8ee9f" }
+foxbox_taxonomy = { git = "https://github.com/ferjm/taxonomy.git", branch = "userid" }
```
Comment (Yoric) [inline Cargo.toml]: I suspect that it would be easier with a `rev`.

**Comment by Yoric** [comment]
r=me, but please don't let the Cargo.toml depend on a (temporary?) branch.

**Comment by ferjm** [comment]
Yeah, we need to land fxbox/taxonomy#55 first :) That's just to make it work locally and to pass the tests on travis. Once fxbox/taxonomy#55 lands, I'll change Cargo.toml to point to taxonomy's latest rev.

Thanks!

**Comment by Yoric** [comment]
I'll let you merge when you're ready.

**Pull request merged**